analytics/server.go

```diff
@@ -206,9 +216,9 @@ func initCloudSQL() (*app, error) {
 
 // Helper - writes Cluster object to Cloud SQL on localhost (via proxy)
 func writeToCloudSQL(app *app, c Cluster) error {
-	sqlInsert := "INSERT INTO CLUSTERS(ClusterId, CreateId, Timestamp, OS, TerraformState, Region, EnableWorkloadIdentity, EnablePreemptibleNodepool, DefaultNodepoolOS, PrivateEndpoint, EnableConfigSync, EnablePolicyController, VPCType, ClusterIndex, ClusterNumNodes, ClusterMachineType, ClusterRegion, ClusterZone) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18)"
+	sqlInsert := "INSERT INTO CLUSTERS3(ClusterId, CreateId, Version, GitCommit, Timestamp, OS, TerraformState, Region, EnableWorkloadIdentity, EnablePreemptibleNodepool, DefaultNodepoolOS, PrivateEndpoint, EnableConfigSync, EnablePolicyController, AnthosServiceMesh, MultiClusterGateway, VPCType, ClusterIndex, ClusterNumNodes, ClusterMachineType, ClusterRegion, ClusterZone) VALUES($1, $2, $3, $4, $5, $6, $7, $8, $9, $10, $11, $12, $13, $14, $15, $16, $17, $18, $19, $20, $21, $22)"
```

Comment:
Should "CLUSTERS3" be "CLUSTERS"

Reply:
no clusters3 is the right table in SQL right now. clusters1 and clusters2 were earlier iterations and I'm not skilled enough in SQL to edit the schemas and edit `clusters` in place. :)